db/migrate/20170512081514_add_join_tables_for_categories.rb

```diff
@@ -0,0 +1,7 @@
+class AddJoinTablesForCategories < ActiveRecord::Migration
+  def change
+    create_join_table :categories, :dramas
+    create_join_table :anime, :categories
+    create_join_table :categories, :manga
```

Comment:
Like all starting with categories first? Cause rails wants it in alphabetical order if using the ```has_and_belongs_to_many``` method http://api.rubyonrails.org/v4.2.7/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_and_belongs_to_many